saleor/graphql/payment/tests/benchmark/test_payment_transactions.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from ....tests.utils import get_graphql_content
+
+NUMBER_OF_ORDERS = 11
+PAYMENT_TRANSACTIONS_QUERY = """
+query {
+  orders(first:100) {
+    edges {
+      node {
+        payments {
+          id
+          gateway
+          isActive
+          created
+          modified
+          token
+          checkout {
+            id
+          }
+          order {
+            id
+          }
+          customerIpAddress
+          actions
+          total {
+            amount
+          }
+          capturedAmount {
+            amount
+          }
+          transactions {
+            id
+          }
+          availableRefundAmount {
+            amount
+          }
+          availableCaptureAmount {
+            amount
+          }
+          creditCard {
+            brand
+          }
+        }
+      }
+    }
+  }
+}
+"""
+
+
+@pytest.mark.django_db
+@pytest.mark.count_queries(autouse=False)
+def test_payment_transactions(
+    staff_api_client, orders_for_benchmarks, permission_manage_orders, count_queries
```

Comment:
We are testing transactions, right? So there are no `Transaction` objects in this test. You need to update the `orders_for_benchmarks` fixture or write the new one or create transactions in test directly.

Reply:
@8r2y5 We should have at least one `Transaction` assigned to the order, and at least one order with multiple transactions. We should also check it in asserts instead of checking order count.